openapi.yaml

```diff
@@ -25694,6 +25707,99 @@ components:
           minimum: 1
           maximum: 100
           example: 6
+        labels:
+          type: object
+          description: |
+            A dictionary of key-value pairs to be added as labels to Node Pool nodes.
+
+            * Specifying an empty dictionary value will remove all previously set labels.
+          additionalProperties:
+            type: object
+            properties:
+              key:
+                type: string
+                minLength: 1
+                maxLength: 63
+                pattern: '^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]${1,63}'
```

Comment:
note for reviewers: I'm not sure about including the pattern as it's specified because it doesn't take into account the third bullet point in the description below, which changes the length and allowed pattern on the prefix.

Reply:
The syntax {1,63} placed after $, I'm not sure that's correct..

Can I remove `${1,63}` from the syntax? 